ome/omero-figure
# Inputs for sliders

See https://trello.com/c/AozomHkL/125-sliders-allow-typing-in-values

For the channel sliders, we now allow users to type in the value they want (as in webclient & Insight).
This allows greater accuracy and you can also enter values outside of the slider range.
We now show the existing Average value (in light grey) if selected images have different values for start/end.
This allows user to edit existing value instead of '-' as before.
We use "number" fields, so you can't ender non-numeric characters. Most browsers also support mousewheel scrolling to edit the value.

To test:
  Select a single image or multiple images:
- Where start/end values for a channel are same in all images, text is black.
- Editing values with slider of by typing in input and hitting enter will update value in all images to the same value (text will become black).
- Can input values outside of slider range. Slider range should expand as needed.
- Check that you can't input a "start" value greater than the "end" value and vice versa (will get alert message on enter).

![screen shot 2016-08-22 at 11 50 42](https://cloud.githubusercontent.com/assets/900055/17853423/46057b7a-6864-11e6-80ec-1eaaab2105c1.png)
